robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -26,7 +26,53 @@ def open_main_menu(self,title):
         self.selenium.wait_until_page_contains(title, 
                                                error=f"{title} link was not found on the page")  
         self.npsp.click_link_with_text(title)
+        locator=npsp_lex_locators["npsp_settings"]["main_menu"].format(title)
+        self.selenium.wait_until_page_contains_element(locator, 
+                                               error=f"click on {title} link was not successful even after 30 seconds")
         
+    def open_sub_link(self,title):  
+        """clicks on the button on the payments page"""  
```

Comment:
Is this a link or button. If it is a link can you please update the documentation accordingly

Reply:
I repurposed a keyword and forgot to update the documentation. Will update it